xml/ha_rear.xml

```diff
@@ -461,73 +443,72 @@
-   and adapt it as needed to create your particular configuration file.
-   Copy options from several example configuration files and paste them
-   into your specific <filename>/etc/rear/local.conf</filename> file that
-   matches your particular system.
-   Do not use original example configuration files, because they provide an
-   overview of variables that can be used for specific setups.
+   Below are some examples of useful configuration options. You can also find example files
+   for different scenarios in the <filename>/usr/share/rear/conf/examples/</filename> subdirectory.
+   Use these to help you create your <filename>/etc/rear/local.conf</filename> file.
   </para>
 
   <example xml:id="ex-ha-rear-nfs-server-backup">
    <title>Using an NFS server to store the file backup</title>
-   <para>
-    &rear; can be used in different scenarios. The following example uses
-    an NFS server as storage for the file backup.
-   </para>
-  </example>
-
-  <procedure>
-   <step>
+  <orderedlist>
+   <listitem>
     <para>
      Set up an NFS server with &yast; as described in the
      <link
       xlink:href="https://documentation.suse.com/sles-15/html/SLES-all/cha-nfs.html">
       &admin; for &sls; &productnumber;</link>.
     </para>
-   </step>
-   <step>
+   </listitem>
+   <listitem>
     <para>
      Define the configuration for your NFS server in the
      <filename>/etc/exports</filename> file. Make sure the directory on the
-     NFS server (where you want the backup data to be available), has the
-     right mount options. For example:
-    </para>
-<screen><replaceable>/srv/nfs</replaceable> *([...],rw,no_root_squash,[...])</screen>
-    <para>
-     Replace <filename>/srv/nfs</filename> with the path to your
-     backup data on the NFS server and adjust the mount options. You might
+     NFS server has the right mount options. For example, you might
      need <literal>no_root_squash</literal> as the
      <command>rear mkbackup</command> command runs as &rootuser;.
+     For more information, see <command>man exports</command>.
     </para>
-   </step>
-   <step>
+   </listitem>
+   <listitem>
     <para>
      Adjust the various <varname>BACKUP</varname> parameters in the
      configuration file <filename>/etc/rear/local.conf</filename> to make
      &rear; store the file backup on the respective NFS server. Find
      examples in your installed system under
      <filename>/usr/share/rear/conf/examples/SLE*-example.conf</filename>.
     </para>
-   </step>
-  </procedure>
+   </listitem>
+  </orderedlist>
+  </example>
+
+  <example xml:id="ex-ha-rear-config-sles-btrfs">
+    <title>Backing up Btrfs subvolumes with <command>tar</command></title>
+    <para>
+     Btrfs is set up with subvolumes, but <command>tar</command>
+     creates backups with the <option>--one-file-system</option> option, which excludes subvolumes.
+     Therefore, the subvolumes must be explicitly included in the &rear; configuration.
```

Comment:
Strictly speaking it is not the actual subvolumes
that need to be specified here but their current
mountpoints - i.e. the directories in the current
filesystem tree via which those btrfs subvolumes
are accessible.
So better wording could be
```
Therefore, the subvolumes mountpoints must be ...
```

E.g. in
https://github.com/rear/rear/wiki/Test-Matrix-rear-2.6#sles-15-sp-1-with-default-btrfs-structure
see what `findmnt -a -t btrfs -o TARGET,SOURCE` shows
```
TARGET                   SOURCE
/                        /dev/sda2[/@/.snapshots/1/snapshot]
|-/var                   /dev/sda2[/@/var]
...
```
The actual btrfs subvolume in the actual btrfs tree is
/@/.snapshots/1/snapshot which is mounted at /
in the current filesystem tree.
The actual btrfs subvolume in the actual btrfs tree is
/@/var which is mounted at /var
in the current filesystem tree.